### DISCUSSION TITLE
jpegli

### DISCUSSION THREAD

**Discussion by iamcarbon** [saucecontrol/PhotoSauce#157] [Ideas]
Some promising metrics here, with an ABI compatible with MozJPEG & libjpeg-turbo. 

https://opensource.googleblog.com/2024/04/introducing-jpegli-new-jpeg-coding-library.html

![image](https://github.com/saucecontrol/PhotoSauce/assets/7537/10dc9392-dea6-4cf1-b4ca-221e038718e0)

**Comment by saucecontrol** [comment]
Yeah, it's exciting stuff!  I've been following jpegli as an [alternative to MozJPEG](https://github.com/saucecontrol/PhotoSauce/issues/109#issuecomment-1475018767) for a while now.

I haven't tested it for perf yet, but it does seem they've worked out the early compatibility issues, so it's probably in a good state to integrate.
